src/vorta/views/archive_tab.py

```diff
@@ -366,6 +379,7 @@ def on_selection_change(self, selected=None, deselected=None):
 
             # special treatment for dynamic mount/unmount button.
             self.bmountarchive_refresh()
+            self.bmountrepo_refresh()
```

Comment:
Why is that needed? I guess it doesn't hurt either.

Reply:
To support changing the repository from the dropdown in the`Repository` tab. Without this, the Unmount option is visible for all repositories when I restart Vorta and not just the one I mounted.

Reply:
I think this would make more sense in `_toggle_all_buttons`